Makefile

```diff
@@ -226,11 +227,15 @@ monitoring_start: docker_check
 docker_loki_install: docker_check
 	docker plugin install grafana/loki-docker-driver:latest --alias loki --grant-all-permissions
 
+clean_mocks:
```

Comment:
Request: Can you change `.PHONY: make docker_loki_install` to `.PHONY: docker_loki_install` above.

Seems like techdebt from the past.